UI/InGameOverworldHelper.cs

```diff
@@ -689,12 +758,27 @@ public static class InGameOverworldHelper {
                     // starting position is all the way left.
                     float x = center.X - (lineWidths[i] / 2f) + 15f;
 
-                    foreach (string tag in lines[i]) {
+                    foreach (CreditsTag tag in lines[i]) {
                         // black edge > BaseColor text background > TextColor tag text
-                        float width = ActiveFont.Measure(tag).X * 0.5f;
-                        Draw.Rect(x - 10, y - 6, width + 20, 44, Color.Black * alphaTags);
-                        Draw.Rect(x - 6, y - 2, width + 12, 36, self.Data.TitleBaseColor * alphaTags);
-                        ActiveFont.Draw(tag, new Vector2(x, y), Vector2.Zero, Vector2.One * 0.5f, self.Data.TitleTextColor * alphaTags);
+                        float width = ActiveFont.Measure(tag.Text).X * 0.5f;
+
+                        if (tag.BorderTexture != null) {
+                            for (int tex_x = 0; tex_x < width + 20; tex_x += tag.BorderTexture.Width)
+                                tag.BorderTexture.Draw(new Vector2(x - 10 + tex_x, y - 6), Vector2.Zero, (tag.BorderColor ?? Color.White) * alphaTags, Vector2.One, 0f,
+                                                       new Rectangle(0, 0, (int) width + 20, 44));
```

Comment:
If I understand correctly, the loop deals with repeating the texture if it is narrower than the tag, and the last parameter here makes the max width of the texture equal to the width of the tag... does that work if the tag is 1.5x the size of the texture?